edx_event_bus_kafka/internal/tests/test_consumer.py

```diff
@@ -130,39 +129,35 @@ def fake_poll(*args, **kwargs):
         ]
         mock_consumer.close.assert_called_once_with()
 
-    def test_emit(self):
-        with patch.object(OpenEdxPublicSignal, 'get_signal_by_type', return_value=self.mock_signal) as mock_lookup:
-            self.event_consumer.emit_signals_from_message(self.normal_message)
+    @patch('edx_event_bus_kafka.internal.consumer.logger', autospec=True)
+    def test_emit(self, mock_logger):
+        self.event_consumer.emit_signals_from_message(self.normal_message)
 
-        mock_lookup.assert_called_once_with(self.signal_type)
+        mock_logger.error.assert_not_called()
         self.mock_signal.send_event.assert_called_once_with(**self.normal_event_data)
 
-    def test_no_type(self):
+    @patch('edx_event_bus_kafka.internal.consumer.logger', autospec=True)
+    def test_no_type(self, mock_logger):
         msg = copy.copy(self.normal_message)
         msg._headers = []  # pylint: disable=protected-access
 
-        with patch.object(OpenEdxPublicSignal, 'get_signal_by_type') as mock_lookup:
-            self.event_consumer.emit_signals_from_message(msg)
-
-        mock_lookup.assert_not_called()
+        self.event_consumer.emit_signals_from_message(msg)
 
-    def test_unknown_type(self):
-        # If we pretend that the test signal type is not a real one, behave accordingly.
-        with patch.object(OpenEdxPublicSignal, 'get_signal_by_type', side_effect=KeyError('not found')) as mock_lookup:
-            # Should just suppress exception and log
-            self.event_consumer.emit_signals_from_message(self.normal_message)
-
-        mock_lookup.assert_called_once_with(self.signal_type)
+        mock_logger.error.assert_called_once_with("Missing ce_type header on message, cannot determine signal")
         assert not self.mock_signal.send_event.called
 
-    def test_unwanted_types(self):
+    @patch('edx_event_bus_kafka.internal.consumer.logger', autospec=True)
+    def test_unwanted_types(self, mock_logger):
```

Comment:
Not a fault with this PR but while we're here, can we rename this test something a little clearer maybe
```suggestion
    def test_wrong_singal_type_in_message_header(self, mock_logger):
```

Reply:
Yeah, absolutely.